lib/src/analytics.dart

```diff
@@ -64,6 +64,10 @@ abstract class Analytics {
 
   /// Boolean that lets the client know if they should display the message
   bool get shouldShowMessage;
+
+  /// Pass a boolean to either enable or disable telemetry and make
+  /// the necessary changes in the persisted configuration file
+  void enableTelemetry(bool reportingBool);
```

Comment:
should this be `setTelemetry`? as it will enable OR disable?

Reply:
I think both work in this case, there was some confusion with the other method for toggling telemetry because of a typo in the documentation